assets/pages/contributions-landing-eu/components/countrySwitcherHeaderContainer.js

```diff
@@ -0,0 +1,48 @@
+// @flow
```

Comment:
This file looks really similar to the countrySwitcherHEaderContainer from US and UK. Is it possible to reduce duplication of code?

Reply:
We discussed this, I think there is potential to reduce the boilerplate for the country switcher. It should be doing the same thing for every contributions page, so it should be possible to extract the logic into a single file. Perhaps it's best to do this when we have all our contributions converted over, because currently it works differently on the old versions.